### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by orowith2os**
Restructures, fixes, and adds some new information.

Would suggest removing the Tested Games section, or ensuring that it's up-to-date.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,203 +46,37 @@ https://discord.gg/6y3BdzC
...
+- Warframe on NVIDIA: you may need to disable GPU Particles in game otherwise the game can freeze randomly. On AMD they work fine
 
 Full patches can be viewed in [protonprep-valve-staging.sh](patches/protonprep-valve-staging.sh).
 
 ## Installation
 
 PLEASE NOTE: There are prerequisites for using this version of proton:
 
-1. You must have vulkan gpu drivers/packages installed properly on your system. VKD3D on AMD requires Mesa 22.0.0 or higher for VK_KHR_dynamic_rendering extension. See https://github.com/lutris/docs/blob/master/InstallingDrivers.md for general driver installation guidance.
+1. You must have the proper Vulkan drivers/packages installed on your system. VKD3D on AMD requires Mesa 22.0.0 or higher for the VK_KHR_dynamic_rendering extension. See https://github.com/lutris/docs/blob/master/InstallingDrivers.md for general driver installation guidance.
```
Comment (Chr1s70ph) [inline README.md:79]: If we are already using hyperlinks e.g. [here in line 60](https://github.com/orowith2os/proton-ge-custom/blob/improve-readme/README.md?plain=1#L60), we could make use of them here too:



```suggestion
1. You must have the proper Vulkan drivers/packages installed on your system. VKD3D on AMD requires Mesa 22.0.0 or higher for the VK_KHR_dynamic_rendering extension. Check [here ](https://github.com/lutris/docs/blob/master/InstallingDrivers.md) for general driver installation guidance.
```

**Review by Chr1s70ph — APPROVED** [review]
Looks good to me 👍🏽

**Comment by orowith2os** [comment]
Any comments on the mfplat section? It might be a bit outdated.

**Comment by tplaiho777** [comment]
I am happy to tell you that Star Wars: Squadron works happily with this Proton release of yours. Very nice.

**Comment by ghost** [comment]
Would you consider removing table of contents part from the file? GitHub automatically generates a table of contents and there is a button to view it.

Screen Shot 2022-09-14 at 9 56 49 AM

**Comment by orowith2os** [comment]
It might be best to keep it, since GitHub's ToC isn't quite as up-front.

**Pull request merged**